server/configs/reload/file_rotate.conf

```diff
@@ -0,0 +1,4 @@
+# Copyright 2018 Authors
```

Comment:
Probably NATS Authors, but I will go through and cleanup in a separate PR.

Reply:
Yeah, this was a placeholder.